frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -183,14 +233,14 @@ class SaveTripModal extends React.Component {
    * @param {Object} tripData Data the new trip document will contain.
    */
   updateExistingTrip(tripId, tripData) {
-    db.collection(COLLECTION_TRIPS)
+    db.collection(DB.COLLECTION_TRIPS)
         .doc(tripId)
         .set(tripData)
         .then(() => {
-          console.log("Document written with ID: ", tripId);
+          console.log('Document written with ID: ', tripId);
         })
         .catch(error => {
-          console.error("Error adding document: ", error);
+          console.error('Error adding document: ', error);
```

Comment:
To add onto the above conversations, I'm unsure if this is safe to leave in for deployed builds? Since leaking error information to users can be potentially dangerous, but yes this might be good to ask Hiram or Longjun about.

Reply:
Yeah that is a good point. I should probably remove the success state `console.log`s altogether and somehow redirect to the error page in error cases like these.

@Hiram-Silvey and @beiebi0 , what is your take on `console.log`ing success states like written tripIds as well as error messages?

Reply:
> I think for the MVP this stuff is OK. Most long-term production systems have separate backend logging for the engineers to inspect. If you tell every user exactly what is happening, 1) it's not very useful for them? 2) they may be able to use it to reverse engineer and break into your system (at least more quickly).

Okay. I will leave it as is and create an issue for removing any `console.log()`s in case we want to be professional after completing MVP.

Reply:
That sounds good. 👍

Reply:
Sweetness.